src/list/ListItem.js

```diff
@@ -190,9 +190,16 @@ const ListItem = props => {
             </Text>
           </View>}
         {textInput &&
-          <View style={[styles.rightTitleContainer, textInputContainerStyle]}>
+          <View
+            style={[
+              styles.rightTitleContainer,
+              { alignItems: null },
```

Comment:
Instead of inlining this here, can you make add it to the styles as textInputContainerStyle?